.travis.yml

```diff
@@ -1,12 +1,18 @@
 sudo: false
-
 language: go
 
 go:
-- 1.9.x
-- 1.10.x
+  - 1.9.x
+  - 1.10.x
 
 go_import_path: github.com/prometheus/tsdb
 
+install:
+  - go get -v -t ./...
+
 script:
-- go test -timeout 5m ./...
+  - go test -timeout 5m ./...
+  # `check_license` target is omitted due to some missing license headers
+  # `staticcheck` target is omitted due to linting errors
+  # `test` target if ommited due to erros while testing with --race
+  - make style unused
```

Comment:
I was getting https://pastebin.com/raw/X75y6njE but found out it's because misbehaving go binary from ubuntu package :/ 

After installing official release it works as expected so a I'm adding the `test` target.